package.json

```diff
@@ -22,7 +22,7 @@
         "test:unit": "jest --testMatch '**/*.test.+(ts|tsx|js)'",
         "test:watch": "jest --watch",
         "version:check": "lerna changed -l -a",
-        "version": "lernacleanchangelog '**' && git add -A",
+        "version": "lernacleanchangelog '**' && git add **/CHANGELOG.md **/package.json package.json pnpm-lock.yaml",
```

Comment:
Er åpen for andre `add`-mønstre her. Det viktigste er at vi _ikke_ adder `.npmrc`-endringer.